### PR TITLE
Add support to install jobs to read cluster status

### PR DESCRIPTION
**Pull request by chaitanyaenr**
Install jobs takes into account cluster status in addition to the
build trigger status to decide whether to run the OCP install or not.
The installs get triggered in cases where there's a new build or if
cluster doesn't exist even if there isn't a new build.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
Depends on https://github.com/openshift-scale/scale-ci-deploy/pull/111.

**Review by mohit-sheth — APPROVED** [review]
LGTM, nice work!

**Comment by mohit-sheth** [comment]
if possible can you update the properties files with CLUSTER_HOST=<jump_host>

File: pipeline-scripts/openshiftv4_on_azure.groovy [hunk 1]
```diff
@@ -106,12 +106,14 @@ stage ('OCP 4.X INSTALL') {
...
 			if ( scale_ci_build_trigger.toBoolean() ) {
-				sh "curl ${scale_ci_build_trigger_url}/payload -o /tmp/payload"
-				sh "curl ${scale_ci_build_trigger_url}/build.status -o /tmp/status"
-				status = readFile "/tmp/status"
-				if ( status.toString().trim().equals("PROCEED") ) {
+				sh "curl ${scale_ci_build_trigger_url}/payload -o /tmp/payload.azure"
+				sh "curl ${scale_ci_build_trigger_url}/build.status -o /tmp/status.azure"
+				sh "curl ${scale_ci_build_trigger_url}/cluster.status -o /tmp/cluster_status.azure"
+				status = readFile "/tmp/status.azure"
+                                cluster_status = readFile "/tmp/cluster_status.azure"
```
Comment (mohit-sheth) [inline pipeline-scripts/openshiftv4_on_azure.groovy:113]: s/spaces/tabs
similarly for others

Comment (chaitanyaenr) [reply pipeline-scripts/openshiftv4_on_azure.groovy]: Fixed it, thanks.

**Comment by chaitanyaenr** [comment]
@mohit-sheth addressed the comments. PTAL.

**Pull request merged**